packages/plugins/typescript/resolvers/src/visitor.ts

```diff
@@ -134,9 +134,12 @@ export class TypeScriptResolversVisitor extends BaseResolversVisitor<
     return `{ ${(node.values || [])
       .map(v => {
         const valueName = v.name as any as string;
-        const mappedValue = valuesMapping[valueName];
+        const mappedValue = valuesMapping[valueName] ?? v.name;
```

Comment:
We are fixing `v.name as any as string` in the next major version so using `valueName` instead of `v.name` here helps reduce conflicts

```suggestion
        const mappedValue = valuesMapping[valueName] ?? valueName;
```